<launch>

    <!-- <node name="PROJECTION_node" pkg="camera_2d_lidar_calibration" type="reprojection.py" output="screen"> -->
    <node name="reprojection" pkg="camera_2d_lidar_calibration" type="reprojection.py" output="screen">
        <param name="scan_topic" value="/euclidean_clustering_pcl2" />
        <!-- <param name="scan_topic" value="/scan" /> -->
        <!-- <param name="image_topic" value="/usb_cam/image_raw" /> -->
        <param name="image_topic" value="/usb_cam/image_rect_color" />
        <param name="calib_file" value="$(find camera_2d_lidar_calibration)/data/calibration_result.txt" />
        <param name="config_file" value="$(find camera_2d_lidar_calibration)/config/config.yaml" />
        <param name="laser_point_radius" type="int" value="3" />    <!-- Reprojected laser point radius size in pixels -->
        <param name="time_diff" type="double" value="1" />  <!-- Maximum time difference in seconds between image and laser scan to be registered and processed -->
    </node>

    <node name="rviz" type="rviz" pkg="rviz" args="-d $(find camera_2d_lidar_calibration)/rviz/reprojection.rviz"/>

</launch>
